pyproject.toml

```diff
@@ -76,7 +76,8 @@ lint.ignore = ["D104", "PLR2004", "RUF200", "S311"]
 
 [tool.ruff.lint.per-file-ignores]
 # S603 unchecked input in subprocess call is fine in our tests
-"*/tests/*" = ["D", "S101", "S603"]
+# PT006 Wrong type passed to first argument of @pytest.mark.parametrize
+"*/tests/*" = ["D", "S101", "S603", "PT006"]
```

Comment:
I had to add this because I am using a fixture within a parametrized test (I think) ... I couldn't find a way around this error so I decided to just ignore it. Happy to dig further into it if anyone thinks it's an issue.

Reply:
@greglucas may know about this

Reply:
I _think_ this might be because it wants it to be spelled out as a list/tuple of options instead of a string.
https://docs.astral.sh/ruff/rules/pytest-parametrize-names-wrong-type/
But I think the string is readable enough so this works IMO and I have also been caught by this error, so it might be better to just ignore it in our tests.

"test_l1a_data, expected_shape" -> ("test_l1a_data", "expected_shape")